docs/vocabulary.md

```diff
@@ -8,27 +8,117 @@ hide_title: true
+
+### BCG Vaccine Products for Intravesical Use​ VS
+​
+Scope/Purpose: Identifies BCG vaccine products for intravesical use
+
+Inclusions: Live BCG for intravesical use
+
+Exclusions: None
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| 90586 | Bacillus Calmette-Guerin vaccine (BCG) for bladder cancer, live, for intravesical use​​ | HCPCS​ | 2019AB | Active
+| ​19 | Bacillus Calmette-Guerin vaccine | CVX | ​ | Active
+
+
+### Confirmed COVID-19 Infection VS
+
+Location: VSAC
+
+OID: [2.16.840.1.113762.1.4.1032.117](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.117/expansion/20200410)
+
+Code Systems: ICD10CM, SNOMEDCT
+
+Type: Grouping
+
+### Convalescent Serum of COVID-19 Products VS
+
+TBD
+
+### Hydroxychloroquine VS
+
+TBD
+
+### Intravesical Administration of BCG Procedure VS
+
+Scope/Purpose: Identifies procedures for intravesical administration of live BCG vaccine. Used to identify patients who received BCG as cellular therapy for bladder cancer.
+
+Inclusions: Procedures specific to the administration of live BCG into the bladder​ (active and inactive terms)
+
+Exclusions: Procedures not specific to BCG​ AND Administration of BCG other than in the intravesical space
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| J9031 | Bcg (intravesical) per instillation​ | HCPCS​ | 2020 | Inactive |
+| J9030​ | Bcg live intravesical instillation, 1 mg​ | HCPCS | 2020​ | Active |
+​301879009​ | Instillation of bacillus Calmette-Guerin into the bladder (procedure)​ | SNOMED CT | 202003​ | Active |
+
+### Intravesical Administration of Therapeutic Agent Procedure VS
+
+Scope/Purpose: Identifies procedures indicative of intravesical administration of therapeutic agent.
+
+Inclusions: Procedures representing the administration of unspecified therapeutic agent into bladder
+
+Exclusions: Procedures specific to BCG (covered by [Intravesical Administration of BCG Procedure VS](#intravesical-administration-of-bcg-procedure-vs))
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| 51720 | Bladder instillation of anticarcinogenic agent (including retention time)​ | CPT​ | 2019AB | Active
+| ​44194005 | Urinary bladder instillation of anticarcinogenic agent (procedure)​ | SNOMED CT | 202003​ | Active
+| ​712753009 | Administration of chemotherapeutic agent into bladder via intravesical route (procedure) | SNOMED CT | 202003​ | Active
+
+### Intravesical Route VS
+
+Scope/Purpose: Represent the administration route directly into urinary bladder
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| ​372471009 | Intravesical route (qualifier)​ | SNOMED CT | 202003​ | Active
+| 261102005 | Intravesical approach (qualifer) | SNOMED CT | 202003​ | Active
+
+### Invasive Mechanical Ventilation VS
```

Comment:
We should add the inclusions and exclusions here.

Reply:
What are they?